pkg/plugin/registry/organization.go

```diff
@@ -18,12 +20,36 @@ func ParseProviderName(name string) (string, string, error) {
 		return strings.ToLower(names[0]), names[1], nil
 	}
 	if len(names) == 1 {
-		return defaultOrganization, name, nil
+		return DefaultOrganization, name, nil
 	}
-	return "", "", fmt.Errorf("invalid provider name %s", name)
+	return "", "", fmt.Errorf("invalid provider name %q", name)
```

Comment:
Why do you use `%q` and not `%w`?

Reply:
Ohh I see, for some reason my brain was assuming you were wrapping the error like you did above
`return fmt.Errorf("could not parse requested provider: %w", err)`